server/resolvers/query/queryResolvers.js

```diff
@@ -101,33 +88,63 @@ module.exports = {
               "fullname",
             ]
           };
-
           const fuse = new Fuse(results.rows, options); // "list" is the item array
           const result = fuse.search(fullnameSearch)
-
           return result
         }
-
         return results.rows
     },
+
+    async githubInfo (parent, {input}, {req, app, postgres}){
+      const userId = authenticate(app, req)
+      const getGithubInfo = {
+        text: 'SELECT github_access_token FROM hired.users WHERE id=$1',
+        values: [userId]
+      }
+      const postgresResponse = await postgres.query(getGithubInfo)
+      const access_token = postgresResponse.rows[0].github_access_token;
+      const result = await axios({
+        url: 'https://api.github.com/graphql',
+        method: 'post',
+        data: {
+          query: `
+            query {
+              viewer{
+                name
+                repositories(last: 1){
+                  nodes {
+                    name
+                    createdAt
+                    updatedAt
+                    description
+                    url
+                    stargazers{
+                      totalCount
+                    }
+                  }
+                }
+              }
+            }
+          `
+        },
+        headers: { 'Authorization': `token ${access_token}` }
+      })
+      return{
+        name: result.data.data.viewer.name,
+        repositories: result.data.data.viewer.repositories.nodes
+      }
+    },
     async listMyDribbbles(parent, _, { app, req, postgres }) {
 			try {
         let userId = authenticate(app, req)
         // let userId = 4
-    
-        console.log('this is userId: ', userId)
-        
 				// getting the userId dribbble_access_token
 				let psql = {
 					text: 'SELECT dribbble_access_token FROM hired.users where id = $1;',
 					values:[userId]
 				}
-				console.log('this is ListMyDribbbles psql', psql)
```

Comment:
Deleted unnecessary console logs which will pollute the console namespace when we're trying to write our own code. Use comments to leave notes if important.